packages/shopping/public/profile.html

```diff
@@ -4,24 +4,32 @@
     <title>
     Shoppy: The #1 online shopping destination
     </title>
+    <meta charset="utf-8">
```

Comment:
Some browsers expect charset to be within the first nth bytes - Let's swap their positions and keep the title in a single line. Let's also add the viewport tag so that it'll render 1x zoom for mobile devices

```suggestion
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Shoppy: The #1 online shopping destination</title>
```